src/services/csv/source.js

```diff
@@ -20,7 +20,7 @@ module.exports = app => {
           'migratedProjectId',
           'createdAt',
           'ownerAddress',
-          'token',
+          'tokenSymbol',
```

Comment:
@mohammadranjbarz  CSV will not work after this

Reply:
> @mohammadranjbarz CSV will not work after this

I'll check it.